msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Overview of publish your app
+description: This is an overview of how to package and publish your app.
+ms.topic: conceptual 
+keywords: teams apps Upload
+---
+
+# Publish your apps
+Apps that you develop need to be published to your organization for the users to use them. There is a complete process to publish your apps that includes, selecting methods of distribution, packaging and uploading, and finally publishing.
+
+This document provides an overview of the steps you must take to publish your app successfully to an App Store or AppSource, formerly Office Store.
+
+## Distribute your apps
+You can distribute your apps to an Enterprise organization, for example, a corporate or US Government organizations, such as federal, state, territorial, and tribal bodies. When distributing the apps, you must ensure that you adhere to the compliance policies of the selected organization. There are different methods to distribute your app. For more information, see [distribute your app](overview.md).
```

Comment:
I think this should be more of a high-level statement how you need to choose your distribution model (individual, team, org, or store).